azure-pipelines-product-construction-service.yml

```diff
@@ -0,0 +1,69 @@
+# Changes the format of the Build.BuildNumber variable
+name: $(Date:yyyyMMdd)$(Rev:r)
+
+trigger:
+  branches:
+    include:
+      - main
+
+pr:
+  branches:
+    include:
+      - main
+
+variables:
+- name: resourceGroupName
+  value: product-construction-service
+- name: containerName
+  value: product-construction-service.api
+- ${{ if ne(variables['Build.SourceBranch'], 'refs/heads/production') }}:
+  - group: Product-Construction-Service-Int
+  - name: containerappName
+    value: product-construction-int
+  - name: containerRegistryName
+    value: productconstructionint
+  - name: containerappEnvironmentName
+    value: product-construction-service-env-int
+  - name: dockerRegistryUrl
+    value: productconstructionint.azurecr.io
+
+stages:
```

Comment:
FYI, you can also have a single-stage YAML, so there's not `stages:` but just
`jobs:` directly at the top level